packages/util-node-http-modules/src/util-node-http-modules.test.js

```diff
@@ -1,6 +1,3 @@
-import {getNodeHttpModules} from "./util-node-http-modules"
-
-test("HTTP Transports exist", async () => {
-  const {nodeHttpsTransport, nodeHttpTransport} = await getNodeHttpModules()
-  expect(nodeHttpTransport && nodeHttpsTransport).toBeDefined()
```

Comment:
Test is removed because experimental ES Modules (`await import(...)`) are not supported in our jest configuration https://jestjs.io/docs/ecmascript-modules.  This test was actually useless in the first place because `expect(null).toBeDefined()` passes and should have been `.toBeTruthy()`